pkg/scheduler/preemption/preemption_test.go

```diff
@@ -1553,7 +1553,7 @@ func TestPreemption(t *testing.T) {
 			}
 			wlInfo := workload.NewInfo(tc.incoming)
 			wlInfo.ClusterQueue = tc.targetCQ
-			targets := preemptor.GetTargets(log, *wlInfo, tc.assignment, snapshotWorkingCopy)
+			_, targets := preemptor.GetTargets(log, *wlInfo, tc.assignment, snapshotWorkingCopy)
```

Comment:
Could you verify if newAssignment is expected?
For now, we can just confirm this equal to `tc.assignment` since we do not enable TAS here.

As as follow-up, it would be better TAS case to this UT, and then verify newAssignment.

Reply:
Refactored in https://github.com/kubernetes-sigs/kueue/commit/f1a3f568c6a062fc5c4491e918957574fb032fa6 which keeps the GetTargets interface the same